server/service/src/backend_plugin/boajs/call_plugin.rs

```diff
@@ -53,28 +60,29 @@ where
     context.run_jobs();
     match promise.state() {
         PromiseState::Fulfilled(JsValue::Undefined) => {}
-        _ => return Err(Error::LoadingModule(name.to_string())),
+        _ => return Err(Error::LoadingModule(r#type.clone())),
     }
 
     // TODO should these be bound as camel case ? Also for inputs and outputs ?
     methods::log::bind_method(context)?;
     methods::sql::bind_method(context)?;
     methods::sql_type::bind_method(context)?;
     methods::get_store_preferences::bind_method(context)?;
+    methods::get_plugin_data::bind_method(context)?;
 
     let namespace = module.namespace(context);
     let plugins = namespace
         .get(js_string!("plugins"), context)?
         .as_object()
         .cloned()
-        .ok_or_else(|| Error::PluginNamespaceMissing(name.to_string()))?;
+        .ok_or_else(|| Error::PluginNamespaceMissing(r#type.clone()))?;
 
-    let key = js_string!(name);
+    let key = js_string!(r#type.as_str());
```

Comment:
How come this one not using `plugin_type_to_string`?

Reply:
it is actually, but naming not great: 70141b844125b750b113cc55d60bc406fb230a57